packages/server/src/app/validation/user.validator.js

```diff
@@ -0,0 +1,56 @@
+const loginSchema = {
```

Comment:
when logging in you don't want to validate the email authenticity and password strength, because that has already been done before.  it is also possible that the validation rules change and then the current users cannot log in

Reply:
Fair point, I'll have a look at it

Reply:
In further inspection, the login schema doesn't validate the email. So I have not changed anything there in that regards, but I did change the validation rules for the password when a user logs in.